### DISCUSSION TITLE
How do I get the tier ID in the webhook payload?

### DISCUSSION THREAD

**Discussion by RaenonX** [community/community#75345] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

What is the value of `sponsorship.tier.node_id` of the webhook event `sponsorship`? I am looking for the value marked as `<TIER_ID>` in the URL below `https://github.com/sponsors/RaenonX/dashboard/tiers/<TIER_ID>/edit`, but I am not sure if `node_id` is that value, since I can't find the sample webhook payload that includes `node_id`, nor sending a test webhook.

**Comment by b8zeek** [comment, marked as answer]
I am pretty sure that `node_id` isn't the same thing as `<TIER_ID>` but GitHub API can help with this; you can use the `node_id` to fetch the sponsorship tier's details, where `<TIER_ID>` should be included.

Good luck. 🍺 

🍀

**Reply by RaenonX** [reply]
Thanks for the response, do you mind help pointing me the query or doc of how to grab the tier ID?

**Reply by b8zeek** [reply]
Not sure exactly as I didn't implement it previously. I would say this is the place to start:

https://docs.github.com/en/graphql/reference/objects#sponsorslisting

Maybe the query can't look something like this:

```
{
  user(login: "YOUR_USERNAME") {
    sponsorsListing {
      tiers(first: 10) {
        edges {
          node {
            id
            name
            description
            monthlyPriceInCents
          }
        }
      }
    }
  }
}
```

I assume you would have to map `id` to the `<TIER_ID>` visible in the URL on the website.

Good luck bro! 🍺 Let us know how this ended for you.

🍀

**Reply by RaenonX** [reply]
Although this doesn't do exactly what I was looking for, but since `node_id` is the same for each tier, and my goal is to just use something unique to tier as reference, I'll just use `node_id`, thanks for this query to allow me pulling tier info tho!

**Reply by b8zeek** [reply]
No worries buddy, I am glad I was able to help.

Take care. 🍺 

🍀

**Reply by ghostinhershell** [reply]
Thanks so much for hopping in here @bejzik8! 🌟

**Reply by b8zeek** [reply]
You're most welcome Mr. Ghost! 👻 You are being so kind to me, thank you very much! <3
